source/_posts/第一阶段总结_李大华.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 2024S 开源操作系统训练营总结-李大华
+date: 2024-11-11 00:18:00
+categories:
+  - 2024秋冬季季开源操作系统训练营
+tags:
+  - author:cndoit18
+
+---
+
+# 第一阶段总结
+
+rustlings很有用，年轻人的第一款cpp.
+
+# 第二阶段总结
+
+老实说，文档很抽象，框架体验也比较一般……
```

Comment:
looking forward to receive your more suggestions.

Reply:
I hope I can learn more content about gdb or cmake,job about OS is hard to seek,but using tools skillfully maybe is very helpful in many other fileds.And I hope I can get more graph in doc,it's helpful to understand it :-)